layouts/default.vue

```diff
@@ -392,9 +307,8 @@ export default {
       console.log('Start NCHAN from mount')
       this.startNCHAN(me.id)
 
-      // Get notifications and chats and poll regularly for new ones.  Would be nice if this was event driven instead but requires server work.
+      // Get chats and poll regularly for new ones.  Would be nice if this was event driven instead but requires server work.
```

Comment:
This comment suggests a timer which has now moved into the store, in notifications.js.  Can you move the comment there?